test/regexp.test.js

```diff
@@ -6,65 +6,66 @@
 // Comment test cases to get CI pass,
 // will recover them when CI config done
 
-// 'use strict';
+'use strict';
 
-// var should = require('should');
-// var db;
+var should = require('should');
+var describe = require('./describe');
+var db;
 
-// describe('cloudant regexp', function() {
-//   var Foo;
-//   var N = 10;
-//   before(function(done) {
-//     db = getSchema();
-//     Foo = db.define('Foo', {
-//       bar: {type: String, index: true},
-//     });
-//     db.automigrate(done);
-//   });
-//   it('create some foo', function(done) {
-//     var foos = Array.apply(null, {length: N}).map(function(n, i) {
-//       return {bar: String.fromCharCode(97 + i)};
-//     });
-//     Foo.create(foos, function(err, entries) {
-//       should.not.exist(err);
-//       entries.should.have.lengthOf(N);
-//       done();
-//     });
-//   });
-//   it('find all foos beginning with b', function(done) {
-//     Foo.find({where: {bar: {regexp: '^b'}}}, function(err, entries) {
-//       if (err) return done(err);
-//       entries.should.have.lengthOf(1);
-//       entries[0].bar.should.equal('b');
-//       done();
-//     });
-//   });
-//   it('find all foos that are case-insensitive B', function(done) {
-//     Foo.find({where: {bar: {regexp: '/B/i'}}}, function(err, entries) {
-//       if (err) return done(err);
-//       entries.should.have.lengthOf(1);
-//       entries[0].bar.should.equal('b');
-//       done();
-//     });
-//   });
-//   it('find all foos like b', function(done) {
-//     Foo.find({where: {bar: {like: 'b'}}}, function(err, entries) {
-//       if (err) return done(err);
-//       entries.should.have.lengthOf(1);
-//       entries[0].bar.should.equal('b');
-//       done();
-//     });
-//   });
-//   it('find all foos not like b', function(done) {
-//     Foo.find({where: {bar: {nlike: 'b'}}}, function(err, entries) {
-//       if (err) return done(err);
-//       entries.should.have.lengthOf(N - 1);
-//       done();
-//     });
-//   });
-//   after(function(done) {
-//     Foo.destroyAll(function() {
-//       done();
-//     });
-//   });
-// });
+describe('cloudant regexp', function() {
+  var Foo;
+  var N = 10;
+  before(function(done) {
+    db = getSchema();
+    Foo = db.define('Foo', {
+      bar: {type: String, index: true},
+    });
+    db.automigrate(done);
+  });
+  it('create some foo', function(done) {
```

Comment:
* Missing `s`; it should be:
`creates some foo`
* Also I would use a more meaningful name than `foo`and `bar`

This comment applies for all your tests

Please feel free to ignore it.